bigtable/docs/snippets.py

```diff
@@ -79,13 +81,24 @@ def setup_module():
     operation = Config.INSTANCE.create(clusters=[cluster])
     # We want to make sure the operation completes.
     operation.result(timeout=100)
+    INSTANCES_TO_DELETE.append(Config.INSTANCE)
 
 
 def teardown_module():
-    Config.INSTANCE.delete()
+    for instance in INSTANCES_TO_DELETE:
+        if instance.exists():
+            instance.delete()
```

Comment:
It would be cleaner to just try deleting the instance and ignore the 404, e.g.:

```python
from google.api_core import exceptions

...

def teardown_module():
    for instance in INSTANCES_TO_DELETE:
        try:
            instance.delete()
        except exceptions.NotFound:
            pass
```